<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCustomFill</class>
 <widget class="QWidget" name="DialogCustomFill">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Custom Fill</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Fill Conditions</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="checkBoxFillToPaths">
        <property name="text">
         <string>Up to and including paths</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxFillToLabels">
        <property name="text">
         <string>Up to other labels</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxFillToUnlabeled">
        <property name="text">
         <string>Up to unlabeled</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxFillToCurvature">
        <property name="text">
         <string>Up to and including different curvature</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxFillToThreashold">
        <property name="text">
         <string>Up to functional values below threshold</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxUseAllPoints">
        <property name="text">
         <string>Use all marked vertices</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Action</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioButtonCreateLabel">
        <property name="text">
         <string>Create new label</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonAddToLabel">
        <property name="text">
         <string>Add to existing label</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonRemoveFromLabel">
        <property name="text">
         <string>Remove from label</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonFill">
       <property name="text">
        <string>Fill</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>DialogCustomFill</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonFill</sender>
   <signal>clicked()</signal>
   <receiver>DialogCustomFill</receiver>
   <slot>OnButtonFill()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>415</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnButtonFill()</slot>
 </slots>
</ui>
